src/flask_restalchemy/api.py

```diff
@@ -257,6 +257,30 @@ def register_view(self, view_func, url, pk="id", pk_type="int", methods=None):
                     f"{url}/<{pk_type}:{pk}>", view_func=view_func, methods=methods
                 )
 
+    def route(self, rule, endpoint=None, **kwargs):
+        """
+        A decorator that is used to register a view function for a
+        given URL rule.  This does the same thing as :meth:`add_url_rule`
+        but is intended for decorator usage::
+
+            @api.route('/')
+            def index():
+                return 'Hello World'
+
+        :param rule: the URL rule as string
+        :param endpoint: the endpoint for the registered URL rule.
+                         Uses the name of the view function by default
+        :param kwargs: the options to be forwarded to the underlying
+                        :meth:`add_url_rule`.
+        """
+
+        def decorator(f):
```

Comment:
Checkout https://realpython.com/primer-on-python-decorators/#who-are-you-really. Good practice for any kind of decorator functions

Reply:
I think in this case there are no such problems because the decorator is only registering the function on the api and then returning the function itself. There is no actual change in the function's behavior.